packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/Dashboard.component.tsx

```diff
@@ -51,7 +51,15 @@ const Dashboard: React.FC<DashboardLayoutProps> = ({ tabs, rancher }) => {
   const { projectId, rancherId } = useParams();
   const { data: versions } = useVersions();
   const { t } = useTranslation('dashboard');
-  useTrackingPage(TrackingPageView.DetailRancher);
+  const { trackPage } = useOvhTracking();
+
+  useEffect(() => {
+    trackPage({
+      pageName: TrackingPageView.DetailRancher,
+      pageType: PageType.dashboard,
+    });
+  }, []);
```

Comment:
I would put this a hook to avoid duplicate :

```tsx
export function usePageTracking(pageName: X, pageType: Y): void {
    const { trackPage } = useOvhTracking();

    useEffect(() => trackPage({pageName, pageType}), [pageName, pageType]);
}
```

And the rules of hooks is not followed

Reply:
Which rules of hooks is not followed ?

It doesn't add much value to create a hook to call a function returned by a hook inside an useEffect

Reply:
you don't have to duplicate this as its already declared in Layout.component.tsx